compile/functions/runtimes/node.js

```diff
@@ -17,6 +18,25 @@ class Node extends BaseRuntime {
     zip.file("package.json", JSON.stringify({main: handlerFile}))
     return zip
   }
+
+  //We're handling a special case here which is that if TypeScript is being used
+  //we won't actually have a ".js" file (this.extension), instead we'll have a "ts"
+  //which should still be considered safe enough, or at least shouldn't
+  //completely stop the deployment process
+  isValidFile(handlerFile) {
+    if (fs.existsSync(handlerFile) || this.isValidTypeScriptFile(handlerFile)) {
```

Comment:
Use `super.isValidFile` rather than `fs.existsSync` directly. This means if we change the super method later on, we won't have to remember to update this method.

Reply:
done!